services/user.py

```diff
@@ -0,0 +1,54 @@
+from django.contrib.auth import get_user_model
+from db.models import User
+from typing import Optional
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str = None,
+    first_name: str = None,
+    last_name: str = None

```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]